### PR TITLE
y2r_u: Cleanup some formatting.

### PR DESCRIPTION
**Pull request by bunnei**
Basically a cleanup to #1447, which was fine, but my OCD got to me. It was also a bit easier for me to just merge and submit a followup PR than overly nitpick, particularly with the volume of PRs we've been getting lately.

### CODE REVIEW

File: src/core/hle/service/y2r_u.cpp [hunk 1]
```diff
@@ -592,32 +620,36 @@ static void SetPackageParameter(Service::Interface* self) {
...
 
     conversion.input_format = params->input_format;
     conversion.output_format = params->output_format;
     conversion.rotation = params->rotation;
     conversion.block_alignment = params->block_alignment;
-    result = conversion.SetInputLineWidth(params->input_line_width);
+
+    ResultCode result = conversion.SetInputLineWidth(params->input_line_width);
+
     if (result.IsError()) goto cleanup;
```
Comment (linkmauve) [inline src/core/hle/service/y2r_u.cpp]: Newline.

**Comment by linkmauve** [comment]
Aside from this nitpick, LGTM.

**Comment by Subv** [comment]
LGTM

**Comment by JamePeng** [comment]
Thanks @bunnei, :)
 I would also like to align the code later……but look like you do it better!

File: src/core/hle/service/y2r_u.cpp [hunk 2]
```diff
@@ -438,15 +453,17 @@ static void GetInputLineWidth(Service::Interface* self) {
...
 }
 
 static void SetInputLines(Service::Interface* self) {
     u32* cmd_buff = Kernel::GetCommandBuffer();
 
-    LOG_DEBUG(Service_Y2R, "called input_lines=%u", cmd_buff[1]);
     cmd_buff[0] = IPC::MakeHeader(0x1C, 1, 0);
     cmd_buff[1] = conversion.SetInputLines(cmd_buff[1]).raw;
+
+    LOG_DEBUG(Service_Y2R, "called input_lines=%u", cmd_buff[1]);
```
Comment (JayFoxRox) [inline src/core/hle/service/y2r_u.cpp]: Does this still work properly after the re-assignment of `cmd_buff[1]`?
(I didn't check the conversion yet)

**Pull request merged**